### PR TITLE
Instr emul s10 v2 pr

### PR DESCRIPTION
**Pull request by KaigeFu**

### CODE REVIEW

File: hypervisor/arch/x86/guest/instr_emul.c [hunk 1]
```diff
@@ -214,12 +214,10 @@ static const struct instr_emul_vie_op one_byte_opcodes[256] = {
...
-static uint64_t size2mask[9] = {
-	[1] = 0xffUL,
-	[2] = 0xffffUL,
-	[4] = 0xffffffffUL,
-	[8] = 0xffffffffffffffffUL,
-};
+static inline uint64_t size2mask(uint8_t bytes)
+{
+	return ((1UL << (bytes << 3U)) - 1UL);
+}
```
Comment (dongyaozu) [inline hypervisor/arch/x86/guest/instr_emul.c]: When bytes = 8, this operation may overflow. So, let us keep use array, but change the oxfffffff to what I mentioned in email,

Comment (dongyaozu) [inline hypervisor/arch/x86/guest/instr_emul.c]: when bytes ==8. it overflows.

Comment (KaigeFu) [reply hypervisor/arch/x86/guest/instr_emul.c]: Ok, will do.

Comment (KaigeFu) [reply hypervisor/arch/x86/guest/instr_emul.c]: @dongyaozu Done.

**Comment by lijinxia** [comment]
Verified+1

**Pull request merged**